src/Compilers/VisualBasic/Portable/Symbols/Source/SourceParameterSymbol.vb

```diff
@@ -332,8 +332,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Dim data = GetEarlyDecodedWellKnownAttributeData()
             If data IsNot Nothing Then
                 Dim attrValue = data.DefaultParameterValue
-                If Not attrValue.IsBad AndAlso
-                    attrValue <> ConstantValue.Unset AndAlso
+                If attrValue <> ConstantValue.Unset AndAlso
```

Comment:
Yes. I'll check it out.

Also, I noticed that some paths still check for `ConstantValue.IsBad`, for example `DecodeDefaultParameterValueAttribute`. I'll check those out too.